lib/components/form/intermediate-place-field.js

```diff
@@ -0,0 +1,112 @@
+import LocationField from '@opentripplanner/location-field'
+import {
+  DropdownContainer,
+  FormGroup,
+  Input,
+  InputGroup,
+  InputGroupAddon,
+  MenuItemA
+} from '@opentripplanner/location-field/lib/styled'
+import React, {Component} from 'react'
+import { connect } from 'react-redux'
+import styled from 'styled-components'
+
+import { clearLocation } from '../../actions/map'
+import { addLocationSearch, getCurrentPosition } from '../../actions/location'
+import { findNearbyStops } from '../../actions/api'
+import { getShowUserSettings } from '../../util/state'
+
+const StyledIntermediatePlace = styled(LocationField)`
+  width: 100%;
+
+  ${DropdownContainer} {
+    display: table-cell;
+    vertical-align: middle;
+    width: 1%;
+  }
+
+  ${FormGroup} {
+    display: table;
+    padding: 6px 12px;
+    width: 100%;
+  }
+
+  ${Input} {
+    display: table-cell;
+    padding: 6px 12px;
+    width: 100%;
+  }
+
+  ${InputGroup} {
+    width: 100%;
+  }
+
+  ${InputGroupAddon} {
+    display: table-cell;
+    vertical-align: middle;
+    width: 1%;
+  }
+
+  ${MenuItemA} {
+    text-decoration: none;
+  }
+
+  ${MenuItemA}:hover {
+    color: #333;
+  }
+`
+
+/**
+ * Component that leverages LocationField to allow selecting an intermediate
+ * place (e.g., stopover on the way from origin to the destination).
+ * TODO: move this to otp-ui?
+ */
+class IntermediatePlaceField extends Component {
+  _removePlace = () => {
+    const {index, location, onLocationCleared} = this.props
+    onLocationCleared && onLocationCleared({location, index})
+  }
+
+  render () {
+    return (
+      <>
+        <StyledIntermediatePlace {...this.props} />
+        <button
+          className='pull-right clear-button-formatting'
+          style={{
+            position: 'relative',
+            top: '-50px',
+            left: '-13px'
+          }}
+          onClick={this._removePlace}>
+          x
+        </button>
```

Comment:
This button overlaps the location-field clear button.

<img width="431" alt="Screen Shot 2020-07-29 at 5 45 26 PM" src="https://user-images.githubusercontent.com/3112493/88867873-8a756a00-d1c3-11ea-86dc-428c921c0686.png">

Reply:
Fixed in 44d4a00